### DISCUSSION TITLE
你也喜歡坐同樣位置嗎?

### DISCUSSION THREAD

**Discussion by Hsucuchia** [1101-GS4719-C/Python-Community-GS4719-C#1222] [General]
進到一個新教室，有人選擇角落的位置、有人選擇靠窗的位置，大家都會挑選自己喜歡的位置坐嗎?
如果挑完自己喜歡的位置後，之後都會想選同個位置坐，當位置被選走後還會有點失落，這是為甚麼呢?
1. 想擁有自己的領域: 人類從以前就會劃分地盤，反覆坐同一個位置就像是發展自己的小型領域，能減少環境帶來的干擾
2. 熟悉會產生安心感: 習慣的座位周遭多半是熟悉的人事物，內心會相對感到較為安心，也有助於提升學習效率
3. 習慣也有助於節省時間: 不用思考要坐哪裡，就能將多出來的時間拿去做其他的事
挑位置的背後有這麼多的知識，因此想要好好學習，挑位置也是重要的因素!

**Comment by GuanYann** [comment]
有時候坐不一樣的位置，上課的視野和氛圍都不一樣了，有時候坐後面看不到，也聽不清楚；坐前面又覺得有點壓迫感，整個感覺都會有所差異，所以固定一個位置對我來說也蠻重要的。

**Comment by zilinng** [comment]
在我的經驗裡，確實上過幾節課之後，就算老師沒有排座位表，大家依然會很有默契的固定自己的座位，我自己也傾向都坐同一個位子，如果坐其他地方會莫名的有點不安，而且不知道為什麼我就是喜歡靠牆坐。

**Comment by DuxtinTai** [comment]
感謝作者的分析，我從來沒有這麼認真想過這個問題，對於我之前喜歡坐在某個位置上課，我一直認定是該位置距離與高度最適合我而已，沒想到背後還有這些幫助。

**Comment by brian431** [comment]
我個人在多數場合都總是坐在最後面邊邊角角的位置，因為實在不喜歡在使用電子產品時附近有人。雖然知道大家也不在乎我在看什麼，不過就是連被餘光瞄到螢幕也不想要。算是我一個奇怪的習慣。

**Comment by Aaron25564448** [comment]
我個人就是完全的第三種狀況，偏好中央偏前方的位置。這種習慣就像吃飯點餐時，比起在那邊選擇性障礙想半天，不如就固定幾種品項輪流點是一樣的概念。

**Comment by Tommy0592** [comment]
我個人習慣在靠角落或靠窗的位置，尤其是在進到新的班級時基本上都離不開這兩種位置，雖然可能有人會覺得坐角落的位置上課不太方便，但是對我來講這個位置可以帶給我莫名的安全感和用電子產品時的隱私感，反而可以讓我更集中注意力上課。

**Comment by KevinCheng42** [comment]
我覺得坐相同的位置真的有助於節省時間，不用花時間思考這次上課要坐哪裡～
因為大家都習慣於坐同樣的位置，所以某次上課時坐前面幾排的同學都沒有來上課，教授就立刻發現他們沒有來～

**Comment by Evening218** [comment]
我自己也是習慣坐在同樣的位置，而且除了覺得熟悉和安心以外，如果去坐其他位置，總有一種侵占他人地盤的感覺。除非我平常是因為搶不到喜歡的座位才不得已坐在固定的地方，那可能就會在有其他座位時換到其他地方坐

**Comment by llh26** [comment]
我個人很喜歡坐在中間偏左或右的位子，太前對我來說有種莫名的壓力，太後方的位子不適合視力不好的人，我也喜歡靠教室兩側的座位，讓人很有安全感！這種舒適感也會提升我上課的品質，減少很多外在環境的干擾，我認為上課能做到喜歡且舒適的位子真的很重要！

**Comment by Chiuhsianglin** [comment]
我認為我自己是第二點，我也喜歡坐在邊邊角角的位置，讓我更有安全感。不會感覺到有太大壓力，對於我上課有很大的幫助

**Comment by ziihui** [comment]
原來有這些原因🥸我也是屬於每次都固定坐在同一個位置的人！
覺得自己是第二點耶～在同一個地方讓我比較有安心感
而且每次都習慣選靠牆，因爲靠在牆壁好舒服哈哈哈哈😂

**Comment by tower20021202** [comment]
我都習慣坐在有認識的人旁邊誒這樣比較有安全感，但若是都沒又認識的人，我就會習慣坐在教室的角落比較不會被發現而且有時候還會有牆可以靠

**Comment by pigeon9975** [comment]
我也喜歡做同樣的位置，除了節省時間外，我覺得熟悉的感覺也很重要。因為視力超差所以我從以前到現在都喜歡坐在前排的位置，這樣不僅看得比較清楚，也比較不容易打嗑睡，我記得之前有一次因為太晚到教室，所以前排的位置都不見了，結果那一堂課我整個超不專心，整個人感覺都不對(？

**Comment by Jack-1D** [comment]
我自己也很喜歡坐在固定的位子上。不只是因為不想額外花時間每次都找不同的位子坐，還是因為不想害其他同學因為我坐了他們的位子而還要花時間再找位子。

**Comment by harrycoconut** [comment]
我從來沒思考過這個問題欸，非常的有趣。以我本身而言，我反倒喜歡每次都挑不一樣的位置，不僅能增加新鮮感，也能每個禮拜認識坐在兩側的新朋友，這樣的習慣也能訓練我更加外向，減低認識陌生人的緊張感😌

**Comment by CCello** [comment]
我喜歡坐在同一個位置，尤其是最後一排，因為後面沒有人比較不會受到其他人的視線影響，而且老師不太會注意到，可以沒有壓力的規劃接下來要做甚麼。

**Comment by YoYooHuang** [comment]
以我的經驗來說，好像大家都有自己一個習慣的模式，久而久之就會習慣坐在同一個位置上。感覺這樣可以得到安全感，而且有一種穩定身心的感覺，不用重新適應陌生的環境。如果是我的話可能會偶爾嘗試坐其他位置，但到最後還是跑回我自己習慣的座位哈哈哈

**Comment by orianna901001** [comment]
每當我到一個新環境時，我通常會去注意牆邊的位置有沒有人坐，就是覺得自己靠牆的感覺會有安心感，感覺就像是背後有人支撐(?)，但是最好背後是沒有人坐，不然我壓力也是很大，感覺後面也會飄來視線。

**Comment by RogelioKG** [comment]
如果一個班級裡有多個朋友群的話，那麼每個人的位置很快就會固定下來，因為朋友都會坐在一塊。
而有時候你生怕坐了其他位置，可能會占到了別人朋友常坐的位置，這或許也是一個固定坐同樣位置的因素。

**Comment by Aaron0810** [comment]
這是真的!!!!雖然在大學中教室上課通常不會規定位子，但只要開學過了兩三個禮拜之後，會發現大家有默契地都坐在同樣的位子上，你總是會可以在同樣的位子發現同樣的人。這跟洗澡一樣，宿舍洗澡我也會等到某一間淋浴間空了才去洗，基本上不太會去使用別間淋浴間，那樣洗起來會相當不自在。

**Comment by Elaine2899** [comment]
如果自己一個去上課的話我剛開始會喜歡嘗試不同的位置，這樣可以觀察不同的視角，直到我找到喜歡的位置；但如果是跟同學一起去上課的話我就會跟著同學一起做相同位置(我發現多數人喜歡做同樣的座位)
